src/DicomMetaDictionary.js

```diff
@@ -155,11 +155,87 @@ class DicomMetaDictionary {
                     });
 
                     naturalDataset[naturalName] = naturalValues;
+                } else if (data.vr === "PN") {
+                    // PN is one of the few deviations from the binary model, see:
+                    // https://dicom.nema.org/dicom/2013/output/chtml/part18/sect_F.2.html
+
+                    // If the PN object was created from a dcm file, it will be a ["string"].
+                    // In this case ValueRepresentation will add a toJSON overload to the
+                    // containing array object. If the object was created from JSON (by
+                    // DICOMWeb for instance), toJSON should work as expected, and the
+                    // denaturalization process will handle the converstion.
+
+                    // This following code provides a consistent accessor experience for
+                    // the naturalized dataset. If the value is a string, it mocks a json
+                    // object, and if it's a json object it mocks a string by overriding
+                    // toString. The latter ensures the ValueRepresentation output is
+                    // correct.
+                    if (
+                        data.Value.__pnDcm ||
+                        (data.Value &&
+                            Array.isArray(data.Value) &&
+                            data.Value[0] == "string")
+                    ) {
+                        data.Value.__objectLike = true;
+                        Object.defineProperty(data.Value, "Alphabetic", {
```

Comment:
I believe the default encoding is as a List of the typed object, and not as a simple object.  However, making the change on the list itself is probably ok, as it makes it easy to access the data.

Reply:
I thought given it was a special case we may as well enforce this "objectlike" behaviour.